src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -130,7 +130,7 @@
     <value>Copy path with quotes</value>
   </data>
   <data name="CopyItemPathWithQuotes" xml:space="preserve">
-    <value>Copy selected item path with quotes</value>
+    <value>Copy item path with quotes</value>
```

Comment:
`selected` helps provide a contrast with the action to copy the path of the current location.

Reply:
To reemphasize, Actions don't have subtitles. This is the label used in tooltips (if we ever expose this option on the toolbar).